frontends/web/src/routes/account/send/components/back.tsx

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright 2024 Shift Crypto AG
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { useNavigate } from 'react-router-dom';
+import { Button, TButton } from '@/components/forms/button';
+
+type TBackButton = Omit<TButton, 'onClick'>;
```

Comment:
you don't actually propagate all the other props to <Button>, so why this type? you only need `children`.

Reply:
good catch it should pass props.. but lets just find out if we want BackButton everywhere or if this is just a temporary component that can be removed once Send is a functional component.